AxonDeepSeg/morphometrics/compute_morphometrics.py

```diff
@@ -45,15 +46,17 @@ def get_pixelsize(path_pixelsize_file):
         return pixelsize
 
 
-def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size=None):
+def get_axon_morphometrics(im_axon, path_folder=None, im_myelin=None, pixel_size=None, circle_approx=True):
```

Comment:
Suggestion: I think setting the variable a boolean and calling it "circle_approx" may not be sufficiently clear to future devs. When the value is false, there's nothing really indicating that it will then be set to ellipse. We could maybe instead call it `axon_shape` and set it to `"circle"` or `"ellipses"`, or something of the sort.

Reply:
Thanks @mathieuboudreau for the suggestion, I agree with the renaming of the variable. I've fixed this in commit 8a42b35